src/pages/jeongeun/Main/Aside.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+const Aside = () => {
+  return (
+    <div className="aside">
+      <aside>
+        <span>
+          {DATA.map((DATA) => {
+            return <span key={DATA.id}>{DATA.content} </span>;
+          })}
```

Comment:
- 매개변수는 UPPER_SNAKE_CASE로 작성할 필요 없음, 보통 camelCase로 작성
- map 메서드의 인자로 들어오는 콜백 함수에서, 로직 없이 단순 return만 한다면 () => () 형태로 작성하여 특별한 로직 없이 JSX 형태만 return하는 것임을 알려줄 수 있음
- 매개변수 자리에서 구조분해할당 가능
```suggestion
          {DATA.map(({ id, content }) => (
            <span key={id}>{content} </span>;
          ))}
```